WordPress/gutenberg
# Reshaping Contributing docs

Adding new docs on how to contribute to WPDS, including an overview, how to get started with contributing, and specific guidance for documentation and components.

Related issue: #66016 

## What?
- New section: Contributing
- New page: Contributing/Overview
- New page: Contributing/Get Ready to Contribute
- New page: Contributing/Documentation
- New page: Contributing/Components

## Why?
Make it easier for anyone to get started contributing to the design system.

## How?
Rewritten pages with clearer structure, an overview page, a new page that explains contributing to documentation, various step-by-step guides, a "how to get started" guide, plus a new "Contributing" section that holds it all.

## Testing Instructions
Please check for inaccuracies, missing content, language.
Also I'd love for someone to check if i edited storybook/preview.js correctly
